### DISCUSSION TITLE
AlpineJS Loading

### DISCUSSION THREAD

**Discussion by nkke30** [alpinejs/alpine#3782] [1. Help] [answered]
No more clowning on git

**Comment by ekwoka** [comment, marked as answer]
Well, Alpine definitely doesn't freeze the page while data is loading, so that needs more explanation.

What do you mean "make that loading via custom html"?

**Reply by ekwoka** [reply]
Well, do you actually have code to do that?

It shouldn't be freezing up unless you did some REALLY wacky stuff. But it would definitely be in your code.

**Reply by ekwoka** [reply]
wow, what the heck is that...

This is really.....a thing...

But your specific problem is that you EXPLICITELY tell jquery to do this synchronously, where you did `async: !1`.

But all of this code is way more convoluted than it needs to be...

I'd recommend dropping jquery and learning actual javascript. jquery is only making your code worse.

**Reply by nkke30** [reply]
🤓
